validator/specs/lib/validateTilesetSpec.js

```diff
@@ -77,3 +78,98 @@ describe('validateTileset', function() {
             }), done).toResolve();
     });
 });
+
+
+describe('validateBoundingVolume', function() {
+   var regionTileset = {
+       "root": {
+           "boundingVolume": {
+               "region": [10, 10, 10, 10, 10, 88]
+           },
+           "content": {
+               "boundingVolume": {
+                   "region":  [0, 0, 0, 0, 20, 30]
```

Comment:
It would make sense to create this tileset so it passes. Right now the content region isn't inside the tile region.